src/components/MessageInput/StopGeneratingButton.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export type StopGeneratingButtonProps = {
```

Comment:
Do we actually need a separate file for this button? Could the markup be inlined into where it is used? As there is only one use of it?

Reply:
Sure thing, makes sense !

Reply:
Since this is now overridable it made sense to keep it separate, as discussed